gender_novels/deployment/static/markdowns/corpus_scraping.md

```diff
@@ -21,15 +21,24 @@ One of the main reasons why websites are against scraping as it causes a drain o
 
 Scraping against the will of a website can have negative repercussions. This could include 'crawler traps', hidden in the website so that algorithms get caught out when trying to scrape. There may also be an IP blocker, that will stop a certain IP from accessing the website. This would be a large problem as we wouldn't be able to do any scraping (including manual scraping) if this were to occur.
```

Comment:
I would say "Scraping against the terms of service of a website" rather than the "will" of a website.